teams/3/settings.html

```diff
@@ -0,0 +1,361 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main>
+      <article>
```

Comment:
Please elaborate more, why we should use div instead of article. Because this is a self-contained part of the page, which can be used in another pages as navigation, we decided to go with article.